solidity/contracts/BancorNetwork.sol

```diff
@@ -1,781 +1,797 @@
-        return convertByPath(_path, _amount, _minReturn, address(0), address(0), 0);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function convert2(
-        address[] memory _path,
-        uint256 _amount,
-        uint256 _minReturn,
-        address _affiliateAccount,
-        uint256 _affiliateFee
-    )
-        public
-        payable
-        returns (uint256)
-    {
-        return convertByPath(_path, _amount, _minReturn, address(0), _affiliateAccount, _affiliateFee);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function convertFor(address[] memory _path, uint256 _amount, uint256 _minReturn, address payable _beneficiary) public payable returns (uint256) {
-        return convertByPath(_path, _amount, _minReturn, _beneficiary, address(0), 0);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function convertFor2(
-        address[] memory _path,
-        uint256 _amount,
-        uint256 _minReturn,
-        address payable _beneficiary,
-        address _affiliateAccount,
-        uint256 _affiliateFee
-    )
-        public
-        payable
-        greaterThanZero(_minReturn)
-        returns (uint256)
-    {
-        return convertByPath(_path, _amount, _minReturn, _beneficiary, _affiliateAccount, _affiliateFee);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function claimAndConvert(address[] memory _path, uint256 _amount, uint256 _minReturn) public returns (uint256) {
-        return convertByPath(_path, _amount, _minReturn, address(0), address(0), 0);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function claimAndConvert2(
-        address[] memory _path,
-        uint256 _amount,
-        uint256 _minReturn,
-        address _affiliateAccount,
-        uint256 _affiliateFee
-    )
-        public
-        returns (uint256)
-    {
-        return convertByPath(_path, _amount, _minReturn, address(0), _affiliateAccount, _affiliateFee);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function claimAndConvertFor(address[] memory _path, uint256 _amount, uint256 _minReturn, address payable _beneficiary) public returns (uint256) {
-        return convertByPath(_path, _amount, _minReturn, _beneficiary, address(0), 0);
-    }
-
-    /**
-      * @dev deprecated, backward compatibility
-    */
-    function claimAndConvertFor2(
-        address[] memory _path,
-        uint256 _amount,
-        uint256 _minReturn,
-        address payable _beneficiary,
-        address _affiliateAccount,
-        uint256 _affiliateFee
-    )
-        public
-        returns (uint256)
-    {
-        return convertByPath(_path, _amount, _minReturn, _beneficiary, _affiliateAccount, _affiliateFee);
-    }
-}
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
```

Comment:
is this only a prettier change?

Reply:
Yeah. Apparently, the old version couldn't parse this file properly, so didn't format it